### ISSUE TITLE
fsa4480.sys error on startup SYSTEM_SERVICE_EXCEPTION

### ISSUE THREAD

**Issue by AnotherJack260** [erdilS/Port-Windows-11-Xiaomi-Pad-5#25]
### Description

Followed the tutorial, on first boot i got an error with this driver.
i tried with older drivers, older driver updater and other windows 11 iso. nothing works

### Reproduction steps

```bash
Followed the tutorial, on first boot i got an error with this driver.
```

### Agreement

- [X] I am using the latest Drivers.
- [X] I am running Windows 11 On The Tablet.
- [X] There is no issue already describing my problem.

**Comment by erdilS** [comment]
If you get fsa4480.sys blue screen, please do this.

1. Remove USB drivers from Driver pack - delete components\QC8150\Device\DEVICE.SOC_QC8150.NABU\Drivers\USB folder.
2. Reinstall drivers via driverupdater.exe - How to update drivers (https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/blob/main/guide/English/update-en.md)
3. After setup reinstall drivers with USB drivers.

**Comment by AnotherJack260** [comment]
thx

**Comment by hyunno0** [comment]
I remove USB drivers from Driver pack - delete components\QC8150\Device\DEVICE.SOC_QC8150.NABU\Drivers\USB folder.
And I installed Windows11 into my MIPAD 5 succesfully...
But, After installing fsa4480.inf Windows went to blue screen...
How can i install USB driver?

**Comment by hyunno0** [comment]
I'm also same. when reinstalling USB driver via fsa4480.inf, Windows went to blue screen and reboot, not installed USB driver...

**Comment by n00b69** [comment]
> when reinstalling USB driver via fsa4480.inf, Windows went to blue screen and reboot, not installed USB driver...

Did you try installing the latest driver package (4 days ago)? This should fix fsa4480.

You need uefi-v3 to use the drivers.

**Comment by hyunno0** [comment]
I try installing the latest driver package (4 days ago)... but blue screen...
And What means "You need uefi-v3 to use the drivers"?

**Comment by hyunno0** [comment]
the uefi-v3 supports huaxing and tianma pannel?
My pannel is  Huaxing...

**Comment by n00b69** [comment]
> the uefi-v3 supports huaxing and tianma pannel? My pannel is Huaxing...

Yes. The UEFI was updated a few months ago to work on both panels. Huaxing or Tiama UEFI images are no longer needed unless you are using very old drivers.

**Comment by hyunno0** [comment]
Thanks 
And I must use only Windows Build: 22631.2861?
I have Windows Build: 22621 ISO file...

**Comment by hyunno0** [comment]
Error installing fsa4480 driver: Publisher in signature catalog not trusted...

**Comment by n00b69** [comment]
> Error installing fsa4480 driver: Publisher in signature catalog not trusted...

Reinstall Windows, you are updating from very old drivers which does not work

**Comment by hyunno0** [comment]
Today, i Reinstalled Windows with 23h2 ...
And i updated new drivers but Error installing fsa4480 driver?...

**Comment by n00b69** [comment]
> Today, i Reinstalled Windows with 23h2 ... And i updated new drivers but Error installing fsa4480 driver?...

Can you join the Telegram at https://t.me/nabuwoa

It will be easier to help there

**Comment by Misha803** [comment]
Can we close this issue ?

**Comment by hyunno0** [comment]
Is the charging issue associated with fsa4480 a perennial task...

**Comment by hyunno0** [comment]
When installing fsa4480 driver, Publisher in signature catalog not trusted too...

**Comment by hyunno0** [comment]
Error installing fsa4480 driver uploaded 17hr ago too : Publisher in signature catalog not trusted...

**Comment by n00b69** [comment]
> Error installing fsa4480 driver uploaded 17hr ago too : Publisher in signature catalog not trusted...

Have you tried [disabling secureboot](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/blob/main/guide/English/disable-secureboot-en.md)?

**Comment by hyunno0** [comment]
In Disabled secureboot, success in installing fsa4480 driver....
But USB HUB(keyboard&mouse) dosn't operate...
Also, when rebooting Windows, Bluescreen...

**Comment by hyunno0** [comment]
When this issue will finish?

**Comment by n00b69** [comment]
Gustave reworks fsa4480, so maybe new driver pack release will be at March 20-22

For now fix of fsa4480 issues is removing fsa4480 line in nabu.xml and reinstalling drivers.
https://github.com/map220v/MiPad5-Drivers/blob/512b6cb849d592d704baef652580d11c1cd6734b/definitions/Desktop/ARM64/Internal/nabu.xml#L29

**Comment by n00b69** [comment]
Extract [this archive](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/releases/download/1.0/fsa4480.7z) to components\QC8150\Device\DEVICE.SOC_QC8150.NABU\Drivers\USB and reinstall drivers.

**Comment by n00b69** [comment]
Try updating this driver manually in device manager

**Comment by hyunno0** [comment]
I updated this driver manually in device manager. 
But USB HUB(keyboard&mouse) dosn't operate...
Also, when rebooting Windows was Bluescreen...
Now, the fsa4480 is instelled as a ON Senmiconductor USB Type-C... on device manager...
What fuction is ON Senmiconductor USB Type-C...?

**Comment by hyunno0** [comment]
FSA4480 is a high performance USB Type-C™ port multimedia switch which supports analog audio headsets. FSA4480 allows the sharing of a common USB Type-C™ port to pass USB2.0 signal, analog audio, sideband use wires and analog microphone signal. FSA4480 also supports high voltage on SBU port and USB port on USB Type-C™ receptacle side.

**Comment by hyunno0** [comment]
After installing new drivers, Bluescreen forever...
Maybe, fsa4480 drivers problem...

**Comment by hyunno0** [comment]
And When re-installing all drivers(except fsa4480 drivers), SLM_Child00 on Device Maneger is Error...

**Comment by n00b69** [comment]
Can you try latest drivers?

**Comment by hyunno0** [comment]
Yes, I did but Bluescreen also...

**Comment by hyunno0** [comment]
Yes, I did, but BlueScreen also...

**Comment by hyunno0** [comment]
How can I download this TESTnabu-drivers.zip?...

**Comment by n00b69** [comment]
USB is in the OEM and Platform folders. Just run OfflineUpdater.cmd with this pack.

**Comment by hyunno0** [comment]
The fsa4480 drivers sent me from you is installing failed...

**Comment by n00b69** [comment]
What does it say?

**Comment by hyunno0** [comment]
Installing the fsa4480 driver on Windows failed...
Do I have to run OfflineUpdater.cmd to reinstall the entire driver...

**Comment by n00b69** [comment]
Try installing it manually first in device manager by selecting the unknown
fsa4480 device

**Comment by hyunno0** [comment]
Try installing it manually in device manager by selecting the unknown fsa4480 device, but failed...

**Comment by n00b69** [comment]
> Try installing it manually in device manager by selecting the unknown fsa4480 device, but failed...

Can you remind me what exactly the issue was again? You don't have working USB, right?

**Comment by hyunno0** [comment]
I re-installed the new drivers on https://github.com/map220v/MiPad5-Drivers...
Which device is unknown to Device Manager?...

**Comment by n00b69** [comment]
> I re-installed the new drivers on https://github.com/map220v/MiPad5-Drivers... Which device is unknown to Device Manager?...

Can you share an image of your device manager?

**Comment by hyunno0** [comment]
![IMG_20240705_195738 1](https://github.com/erdilS/Port-Windows-11-Xiaomi-Pad-5/assets/140943156/f3cae23e-1e84-450d-ae33-29b96696d60f)

**Comment by n00b69** [comment]
USB works in which version of the drivers?

**Comment by hyunno0** [comment]
![image](https://github.com/user-attachments/assets/df8401b5-d758-40bf-bad0-436dd8e5ed41)

**Comment by hyunno0** [comment]
When I re-installed the new drivers, The above message appears...
And USB not works...

**Comment by hyunno0** [comment]
![image](https://github.com/user-attachments/assets/ee10722d-5497-452f-90b3-7aaa626ed307)

Also the same error...

**Comment by hyunno0** [comment]
![image](https://github.com/user-attachments/assets/93e717e3-90cb-4a14-8ddd-e7fcd17a5a18)

When I installed the old drivers, There is no error...

**Comment by n00b69** [comment]
The error in the drivers does not matter, it is related to app packages.

USB works in old drivers?

**Comment by hyunno0** [comment]
Yes, USB works in old drivers but not works in latest drivers...

**Comment by hyunno0** [comment]
[CODE_OF_CONDUCT.md](https://github.com/user-attachments/files/16319416/CODE_OF_CONDUCT.md)
[ETW.md](https://github.com/user-attachments/files/16319417/ETW.md)
[LICENSE.md](https://github.com/user-attachments/files/16319418/LICENSE.md)
[NTDBG.md](https://github.com/user-attachments/files/16319419/NTDBG.md)
[README.md](https://github.com/user-attachments/files/16319420/README.md)

Here...

**Comment by n00b69** [comment]
Do you have the original zip file by any chance?

**Comment by hyunno0** [comment]
Why not USB works in latest drivers in my case...
can you check this issue...
And What is my technical support?

**Comment by n00b69** [comment]
Can you try (if possible) a clean Windows install with the drivers I linked? If they work, I can ask the developer of the drivers to look into it

**Comment by hyunno0** [comment]
There is a lot of problems in USB...
In usb host mode, not charging by the PD of USB HUB but only can charge by Charging Adapter...
In none usb host mode, USB HUB not be recognized...
We hope these things will be improved as soon as possible...

**Comment by n00b69** [comment]
Everyone has that issue and it sadly cannot be fixed.

**Comment by hyunno0** [comment]
It cannot be fixed forever? or have possibility next time?

**Comment by hyunno0** [comment]
This issue be never fixed forever?...

**Comment by n00b69** [comment]
Unlikely

**Comment by NewbieHerePlsHelp** [comment]
![Uploading IMG_20240825_155643.jpg…]()
Somebody help me I don't know what to do in BSOD in my pad 5, I don't know how to do the code thing🥺

**Comment by n00b69** [comment]
Try these drivers

https://github.com/map220v/MiPad5-Drivers/releases/tag/2024.02.29-2

**Comment by NewbieHerePlsHelp** [comment]
Thanks
    On Monday, August 26, 2024 at 08:00:01 AM GMT+2, the sog ***@***.***> wrote:  
 
 


Try these drivers

https://github.com/map220v/MiPad5-Drivers/releases/tag/2024.02.29-2

**Comment by hyunno0** [comment]
When the latest Drives will post?...

**Comment by n00b69** [comment]
There are no new drivers, but please tell me if the drivers I linked above work

**Comment by hyunno0** [comment]
there's no update from the drivers...

**Comment by n00b69** [comment]
This should be fixed in latest drivers and UEFI, reopen the issue or make a new one if it isn't
